Feature/prepare json yaml files (#5)
nimh-dsst/nda-bids-upload: README.md

> ## :mega: Repository No Longer Being Maintained :mega:
> 
> **We have adopted a new process for storing and sharing ABCD data and will therefore no longer be maintaining this repository!**

Author: ```suggestion
## :mega: Repository No Longer Being Maintained by DCAN-Labs :mega:

**DCAN-Labs has adopted a new process for storing and sharing ABCD data and will therefore no longer be maintaining this repository!**
```

> # Welcome to the DCAN Labs NDA BIDS Upload Repository

Author: ```suggestion
# Welcome to the NDA BIDS Upload Repository
```
